tests/integration/test_deletes.py

```diff
@@ -291,7 +291,7 @@ def test_partitioned_table_positional_deletes_sequence_number(spark: SparkSessio
     assert snapshots[2].summary == Summary(
         Operation.OVERWRITE,
         **{
-            "added-files-size": "1145",
+            "added-files-size": "1125",
```

Comment:
Shall we also update this one to something like
```python
snapshots[2].summary["added-files-size"]
```
or this is intentionally left in hard-coded version?

Reply:
Eagle eye, thanks!